### ISSUE TITLE
Problems with error handling when extending external API

### ISSUE THREAD

**Issue by renestalder** [APIs-guru/graphql-faker#70]
So as an example, I have something like this.

```sh
graphql-faker -p 3002 -e https://myapi.tld/graphql
```

And a query like this:

```
query ValidateEmail($email: String!) {
  validateEmail(email: $email)
}
```

This query can either return this result:

```json
{
  "data": {
    "validateEmail": true
  }
}
```

Or in the error case, something like this:

```
{
  "errors": [
    {
      "message": "20006",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "validateEmail"
      ],
      "extensions": {
        "code": "INTERNAL_SERVER_ERROR",
        "exception": {
          "stacktrace": [
            "Error: 20006",
            "    at CustomerResolver.<anonymous> (/app/dist/customer/customer.resolver.js:55:27)",
            "    at Generator.next (<anonymous>)",
            "    at fulfilled (/app/dist/customer/customer.resolver.js:16:58)",
            "    at process._tickCallback (internal/process/next_tick.js:68:7)"
          ]
        }
      }
    }
  ],
  "data": null
}
```

Now the problem seems that graphql-faker can't handle the error response. Apollo Client gives you something like that:

```
[frontend] 10:32:47 [GraphQL error]: Message: Cannot return null for non-nullable field Query.validateEmail., Location: [object Object], Path: validateEmail
[frontend] 10:32:47 [Network error]: ServerError: Response not successful: Received status code 500
```

Which only appears when using graphql-faker in between to extend the real API. Is there any solution to this?

**Comment by IvanGoncharov** [comment]
@renestalder Do you see this error messages in console:
https://github.com/APIs-guru/graphql-faker/blob/dd0dcb466ef294613318daa63ac1d200b9ce596d/src/proxy.ts#L73
If so I will try to fix it in the evening.
If not please provide more details so I could reproduce this issue.

**Comment by renestalder** [comment]
@IvanGoncharov It's not coming with Global Errors prefixed.

I'll try to recreate that with a public API to give you an example case. Very much possible it will turn out not to be a problem with GraphQL faker, it was just most possible as switching to the real API without graphql-faker in between did resolve the issue.

**Comment by IvanGoncharov** [comment]
@renestalder Fixed by @niekert and released as 1.9.2 📦
Can you please test it?

**Comment by renestalder** [comment]
@IvanGoncharov @niekert Woah cool, thanks you so much guys. Going to give it a shot next week.
